libs/test/SalesforceSDKTest/src/com/salesforce/androidsdk/rest/RestRequestTest.java

```diff
@@ -31,23 +31,28 @@
 
 import junit.framework.TestCase;
```

Comment:
These tests do not go to the server - they simply look at the request object built.